docker-compose.yml

```diff
@@ -1,51 +1,44 @@
-version: "3.8"
+version: '3.8'
 services:
   api:
-    container_name: api
-    image: mdm-api
     build:
-      dockerfile: Dockerfile
       context: .
-      target: development
-    volumes:
-      - .:/app/src:rw
-    env_file:
-      - .env
-    environment:
-      PORT: 3003
-      DATABASE_MDM_HOST: ${DATABASE_MDM_HOST}
-      DATABASE_MDM_PORT: ${DATABASE_MDM_PORT}
-      DATABASE_MDM_USERNAME: ${DATABASE_MDM_USERNAME}
-      DATABASE_MDM_PASSWORD: ${DATABASE_MDM_PASSWORD}
-      DATABASE_MDM_NAME: ${DATABASE_MDM_NAME}
-      DATABASE_NUMBER_GENERATOR_HOST: ${DATABASE_NUMBER_GENERATOR_HOST}
-      DATABASE_NUMBER_GENERATOR_PORT: ${DATABASE_NUMBER_GENERATOR_PORT}
-      DATABASE_NUMBER_GENERATOR_USERNAME: ${DATABASE_NUMBER_GENERATOR_USERNAME}
-      DATABASE_NUMBER_GENERATOR_PASSWORD: ${DATABASE_NUMBER_GENERATOR_PASSWORD}
-      DATABASE_NUMBER_GENERATOR_NAME: ${DATABASE_NUMBER_GENERATOR_NAME}
-      DATABASE_CEDAR_HOST: ${DATABASE_CEDAR_HOST}
-      DATABASE_CEDAR_PORT: ${DATABASE_CEDAR_PORT}
-      DATABASE_CEDAR_USERNAME: ${DATABASE_CEDAR_USERNAME}
-      DATABASE_CEDAR_PASSWORD: ${DATABASE_CEDAR_PASSWORD}
-      DATABASE_CEDAR_NAME: ${DATABASE_CEDAR_NAME}
-      DATABASE_CIS_HOST: ${DATABASE_CIS_HOST}
-      DATABASE_CIS_PORT: ${DATABASE_CIS_PORT}
-      DATABASE_CIS_USERNAME: ${DATABASE_CIS_USERNAME}
-      DATABASE_CIS_PASSWORD: ${DATABASE_CIS_PASSWORD}
-      DATABASE_CIS_NAME: ${DATABASE_CIS_NAME}
-      REDIS_HOST: ${REDIS_HOST}
-      REDIS_PORT: ${REDIS_PORT}
-      SWAGGER_PASSWORD: ${SWAGGER_PASSWORD}
-      SWAGGER_USER: ${SWAGGER_USER}
-      NODE_ENV: development
-      TZ: Europe/London
-    command: npm run start:dev
+    image: api
+    container_name: api
     restart: always
+    command: npm run start:prod
+    extra_hosts:
+      - "${DATABASE_MDM_HOST}:${DATABASE_MDM_IP}"
+      - "${DATABASE_CEDAR_HOST}:${DATABASE_CEDAR_IP}"
     ports:
-      - 3003:3003
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:3003"]
-      interval: 1m30s
-      timeout: 10s
-      retries: 3
-      start_period: 40s
```

Comment:
Do we not need this anymore?

Reply:
I have removed this, healthcheck will be added natively in Azure.